### PR TITLE
// Checkout cleanup

### PR DESCRIPTION
**Pull request by ishcherbakov**

### CODE REVIEW

File: themes/classic/_dev/css/modules/checkout.scss [hunk 1]
```diff
@@ -1,177 +1,136 @@
...
-    
-     .content {
-        padding: 30px;
-    }
-    
-    label {
-        font-weight: 700;
-        text-align: left;
+  &.-js-current {
+    .content {
```
Comment (nihco2) [inline themes/classic/_dev/css/modules/checkout.scss]: Please don't style class prefixed by js-, it's only for javascript use

Comment (ishcherbakov) [inline themes/classic/_dev/css/modules/checkout.scss]: It's not related to current PR. I've just normalised indents in SCSS file

Comment (nihco2) [inline themes/classic/_dev/css/modules/checkout.scss]: Ok, I'll clean that later, thx.

File: themes/classic/templates/checkout/_partials/address-form.tpl [hunk 2]
```diff
@@ -11,29 +11,26 @@
...
-        <input  name    = "use_same_address"
-                type    = "checkbox"
-                value   = "1"
-                {if $use_same_address} checked {/if}
-        >
-        <span>{l s='Use this address for invoice too'}</span>
-      </label>
-  </div>
+    <div class="form-group row">
+      <div class="col-md-3"></div>
```
Comment (maximebiloe) [inline themes/classic/templates/checkout/_partials/address-form.tpl]: Why do you use an empty `div` and not a `col-md-offset-3`?

Comment (ishcherbakov) [inline themes/classic/templates/checkout/_partials/address-form.tpl]: Fixed

**Pull request merged**